docs/en/index.md

```diff
@@ -2,21 +2,57 @@
 title: Home
 ---
 
-## A simple extensible CSS framework, written in Sass.
+## Get started
 
-[![Build Status](https://travis-ci.org/vanilla-framework/vanilla-framework.svg?branch=master)](https://travis-ci.org/vanilla-framework/vanilla-framework)
-[![npm version](https://badge.fury.io/js/vanilla-framework.svg)](http://badge.fury.io/js/vanilla-framework)
-[![Downloads](https://img.shields.io/npm/dm/vanilla-framework.svg)](https://www.npmjs.com/package/vanilla-framework)
-[![devDependency Status](https://david-dm.org/vanilla-framework/vanilla-framework/dev-status.svg)](https://david-dm.org/vanilla-framework/vanilla-framework#info=devDependencies)
-[![Chat in #vanilla-framework on Freenode](https://img.shields.io/badge/chat-%23vanilla--framework-blue.svg)](http://webchat.freenode.net/?channels=vanilla-framework)
+You can use Vanilla in your projects in a few different ways.
 
-Vanilla Framework contains a basic CSS grid and pattern classes, and is designed to be extended either directly or by creating extension themes.
+### Install via npm
 
-Sites created within Canonical should follow this style guide closely, whereas external sites are free to adapt and expand the existing components to their needs.
+The recommended way to get Vanilla is through npm:
 
-For ways to use Vanilla framework, see [vanilla-framework/vanilla-framework on GitHub](https://github.com/vanilla-framework/vanilla-framework).
+`npm install --save vanilla-framework`
```

Comment:
I commented a long time back on the Google doc that this should be `--save` as VF is a runtime dependency, I seem to remember @nottrobin concurring but it seems to have been lost again in the Google doc.